src/scene/renderer/renderer.js

```diff
@@ -505,6 +505,8 @@ class Renderer {
         // Alpha test
         if (material.opacityMap) {
             this.opacityMapId.setValue(material.opacityMap);
+        }
+        if (material.opacityMap || material.alphaTest > 0) {
```

Comment:
So if the following material has alphaTest === 0, it won't be set and will keep using the previously set value.

Reply:
If there is no opacity map and alphaTest == 0, then there will be no alpha test shader code?